settings/cronjob.ini.append.php

```diff
@@ -0,0 +1,12 @@
+<?php /*
+
+[CronjobSettings]
+ExtensionDirectories[]=ezplatformsearch
+
+[CronjobPart-frequent]
```

Comment:
And I always removed it from frequent part :D

Adding a note to README is okay, ofcourse.